README.md

```diff
@@ -1,26 +1,20 @@
 # SmartWeave SDK v2
 
-#### Warning: SDK is currently in alpha version.
-#### We do not recommend using it in a production env.
-
-#### Features not yet implemented
-- deploying contracts
-- arTransfer and tags handling for "viewState" ("interactRead")
-- "dry-runs" (though not sure if this should be part of the "core" SDK)
-- CLI (though not sure if that is a necessary - even if, it should be 
-probably a separate lib built on top of the base SDK).
-
 SmartWeave SDK v2 is the new, rewritten from scratch, SDK implementation proposal for interaction with SmartWeave Contracts.
 It has been built with performance (e.g. caching at multiple layers, Arweave calls optimization)
 and modularity (e.g. ability to use different types of caches, imported from external libraries) in mind.
 
-Please consider this as a work in progress and use at your own risk :-).
+#### Warning: SDK is currently in alpha version.
+We're already using the new SDK on production, both in our webapp and nodes.
+If you would like to use it in production env, please contact us at our discord.
```

Comment:
However, if you'd like to use it in production as well, please contact us on [discord](https://discord.com/invite/PVxBZKFr46) to ensure a smooth transition and get help with testing